src/test/java/core/basesyntax/service/ParseServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.template.FruitTransaction;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParseServiceImplTest {
+    private static ParseService parseService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        parseService = new ParseServiceImpl();
+    }
+
+    @Test
+    public void parseLineNullInputLine_NotOk() {
+        String line = null;
+
+        try {
+            parseService.parseLine(line);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown if value is null");
+    }
+
+    @Test
+    public void parseLineEmptyInputLine_NotOk() {
+        String line = null;
+
+        try {
+            parseService.parseLine(line);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown if value is empty");
+    }
+
+    @Test
+    public void parseLineIncorectInputLine_1_NotOk() {
```

Comment:
specify why it is incorrect instead of numbering tests. how should people know what's tested without reading the test?

Reply:
> specify why it is incorrect instead of numbering tests. how should people know what's tested without reading the test?

fixed)